tools/refactoring/check-copyright.py

```diff
@@ -0,0 +1,68 @@
+import sys 
+import os
+import re
+import codecs
+
+import refactoring # Note: refactoring.py need to be in the current working directory
+
+try:
+	import colorama
+	colorama.init()
+	
+	def print_error(mess, **kwargs):
+		print(colorama.Fore.RED + colorama.Style.BRIGHT + mess + colorama.Style.RESET_ALL, **kwargs)
+	def print_warn(mess, **kwargs):
+		print(colorama.Fore.YELLOW + colorama.Style.BRIGHT + mess + colorama.Style.RESET_ALL, **kwargs)
+except ImportError:
+	def print_error(mess, **kwargs):
+		print(mess, **kwargs)
+	def print_warn(mess, **kwargs):
+		print(mess, **kwargs)
+
+paths = sys.argv[1:]
+
+
+excludespatterns = ["*/ext/*", "*moc_*", "*/proteindocking/*", "*/proteindocking2/*", 
+					"*/genetree/*", "*.DS_Store", "*DS_mapp", ".md", "*.suo" , "*.h5",
+					"*.jpg", "*.JPG", "*.jpeg", "*.lib", "*.dll", "*.inv",
+					"*.png", "*.ttf", "*.tif", "*.pyc", "*.raw", "*.bmp", "*.wav", "*.xcf", "*.ico", "*.icns",
+					"*.qch", "*.qhc", "*.exr", "*.pwm", "*.pvm", "*.pdf", "*.otf", "*.exe", "*.fbx"]
+
+files = refactoring.find_files(paths, ['*'], excludes=excludespatterns)
+
+def replace(pattern, replacement) :
+	matches = refactoring.find_matches(files, pattern)
+	print(matches.size());
+
+
+copyright_regex = [
+	re.compile(r"(\s*[*#]\s+Copyright \(c\) 201\d-)201[1234567]( Inviwo Foundation\s*)") , 
```

Comment:
Is it possible to get the current year so that we do not  need to update the script each year?

Reply:
See comment on PR